### PR TITLE
UI Cleanup

### PR DESCRIPTION
**Pull request by jeznorth**
## Description of Changes

- Various changes to front-end to clean up disparate patterns and UI elements.

### CODE REVIEW

File: app/src/components/attachments/list/AttachmentsList.tsx [hunk 1]
```diff
@@ -90,30 +61,33 @@ const AttachmentsList = <T extends IGetProjectAttachment | IGetSurveyAttachment>
...
-              <TableRow key={`${attachment.fileName}-${attachment.id}`}>
-                <TableCell scope="row" className={classes.attachmentNameCol}>
-                  <Box display="flex" alignItems="center">
-                    <Icon
-                      path={icon}
-                      size={1}
-                      className={classes.fileIcon}
-                      style={{ marginRight: '16px', marginLeft: '4px' }}
-                    />
+              <TableRow hover={false} key={`${attachment.fileName}-${attachment.id}`}>
```
Comment (curtisupshall) [inline app/src/components/attachments/list/AttachmentsList.tsx:64]: Super nit: we can just use `attachment.id` as the key here

File: app/src/components/publish/PublishSurveyDialog.tsx [hunk 2]
```diff
@@ -146,11 +146,14 @@ const PublishSurveyDialog = (props: IPublishSurveyIdDialogProps) => {
...
               </DialogContent>
               <DialogActions>
-                <Typography component="span" variant="subtitle2" fontWeight="700" sx={{ mr: 2 }}>
+                <Typography component="span" variant="subtitle2" sx={{ mr: 2 }}>
                   <Typography component="span" color="textSecondary" variant="inherit">
-                    Last Published:
-                  </Typography>{' '}
-                  {publishDate}
+                    {publishDate ? (
+                      <span>Status:&nbsp;&nbsp;<b>Published ({publishDate})</b></span>
```
Comment (curtisupshall) [inline app/src/components/publish/PublishSurveyDialog.tsx]: Do we prefer using `<strong>` over `<b>`?

**Review by curtisupshall — COMMENTED** [review]
Code looks good to me! Just had a couple minor comments

File: app/src/components/publish/PublishSurveyDialog.tsx (hunk 2 above)
Comment (jeznorth) [reply app/src/components/publish/PublishSurveyDialog.tsx]: No difference here really.

File: app/src/components/attachments/list/AttachmentsList.tsx (hunk 1 above)
Comment (jeznorth) [reply app/src/components/attachments/list/AttachmentsList.tsx:64]: I don't see why not. I'll update

**Review by KjartanE — DISMISSED** [review]
Tested locally, looks good!

**Review by curtisupshall — APPROVED** [review]

**Review by KjartanE — APPROVED** [review]
🐈

**Pull request merged**